### PR TITLE
SSD-3745 add max value to numbers

### PR DESCRIPTION
**Pull request by fabian202**

### CODE REVIEW

**Review by cesar82 — APPROVED** [review]

File: src/components/pages/ProgramDetailsPage/schema/index.js [hunk 1]
```diff
@@ -4,6 +4,9 @@ import { ONE_TIME_DURATION } from 'utils/constants'
 import * as criteria from './criteria'
 import * as sections from './sections'
 
+// const maxNumber = 99999999999999
```
Comment (normandanielmq) [inline src/components/pages/ProgramDetailsPage/schema/index.js:7]: Remove unused code

File: src/components/pages/ProgramDetailsPage/schema/index.js [hunk 2]
```diff
@@ -4,6 +4,9 @@ import { ONE_TIME_DURATION } from 'utils/constants'
...
+const maxNumber = 1e+14
```
Comment (normandanielmq) [inline src/components/pages/ProgramDetailsPage/schema/index.js:8]: Has javascript a function for this?

Comment (normandanielmq) [reply src/components/pages/ProgramDetailsPage/schema/index.js:8]: `Number.MAX_SAFE_INTEGER`

Comment (fabian202) [reply src/components/pages/ProgramDetailsPage/schema/index.js:8]: Thanks, gonna try this one

**Review by normandanielmq — APPROVED** [review]

**Pull request merged**